src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj

```diff
@@ -23,14 +23,11 @@
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="$(MicrosoftAspNetCoreHostingAbstractionsPackageVersion)" />
   </ItemGroup>
 
-  <ItemGroup>
-    <SignedPackageFile Include="$(TargetPath)" PackagePath="lib/netstandard2.0/Microsoft.AspNetCore.Server.IIS.dll" Certificate="Microsoft" />
-  </ItemGroup>
-
   <ItemGroup Condition="$(PackNativeAssets) == 'true'">
     <SignedPackageFile Include="aspnetcorev2_inprocess_x86" PackagePath="runtimes/win-x86/nativeassets/$(TargetFramework)/aspnetcorev2_inprocess.dll" Certificate="Microsoft" />
     <SignedPackageFile Include="aspnetcorev2_inprocess_x64" PackagePath="runtimes/win-x64/nativeassets/$(TargetFramework)/aspnetcorev2_inprocess.dll" Certificate="Microsoft" />
 
+    <Content Include="$(PackageId).targets" PackagePath="build/$(TargetFramework)/" />
```

Comment:
Why was this added?

Reply:
The nuspec contained this: https://github.com/aspnet/IISIntegration/blob/182299235438d9272066f0b34fd5f93d24bbe1de/src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.nuspec#L27. Do we still need this targets file? 

https://github.com/aspnet/IISIntegration/blob/182299235438d9272066f0b34fd5f93d24bbe1de/src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.targets#L1-L9

Reply:
Yeah. Makes sense now 👍